e2e/tests/session.feature

```diff
@@ -5,7 +5,7 @@ Feature: Session Namespace
   Scenario: Supplier completes claim/proof lifecycle for a valid session
     Given the user has the pocketd binary installed
     When the supplier "supplier1" has serviced a session with "5" relays for service "svc1" for application "app1"
-    And the user should wait for "5" seconds
+    And the user should wait for "7" seconds
```

Comment:
```suggestion
    # TODO_TECHDEBT: Once the session grace period is configurable, set it to 0
    # at the beginning of the test to reduce this wait time as much as possible.
    And the user should wait for "7" seconds
```

Reply:
I don't understand how this relates to session grace period configuration. To me it's the time we need to wait before querying for claims and see if they incremented. I believe it mostly serves as a timeout.